ResearchKit/Common/ORKResult.h

```diff
@@ -656,6 +656,74 @@ ORK_CLASS_AVAILABLE
 
 @end
 
+/**
+ The `ORKGoNoGoSample` class represents a single reading from the accelerometer.
+ 
+ The gonogo sample object records the time and magnitude of the acceleration. 
+ A gonogo sample is included in an `ORKGoNoGoResult` object, and is recorded by the
+ step view controller for the corresponding task.
+ */
+ORK_CLASS_AVAILABLE
+@interface ORKGoNoGoSample : NSObject <NSCopying, NSSecureCoding>
+
+/**
+ A relative timestamp indicating the time of the accelerometer event.
+ 
+ The timestamp is relative to the time the stimulus was displayed
+ */
+@property (nonatomic, assign) NSTimeInterval timestamp;
+
+/**
+ Magnitude of the acceleration event.
+ */
+@property (nonatomic, assign) double vectorMagnitude;
+
+@end
+
+/**
+ The `ORKGoNoGoResult` class represents the result of a single successful attempt within an ORKGoNoGoStep.
+ 
+ The `timestamp` property is equal to the value of systemUptime (in NSProcessInfo) when the stimulus occurred.
+ Each entry of motion data in this file contains a time interval which may be directly compared to timestamp in order to determine the elapsed time since the stimulus.
+ 
+ The fileResult property references the motion data recorded from the beginning of the attempt until the threshold acceleration was reached.
+ Using the time taken to reach the threshold acceleration as the reaction time of a participant will yield a rather crude measurement. Rather, you should devise your 
+ own method using the data recorded to obtain an accurate approximation of the true reaction time. Still, the time is provided in time to threshold
+ 
+ A reaction time result is typically generated by the framework as the task proceeds. When the task
+ completes, it may be appropriate to serialize the sample for transmission to a server
+ or to immediately perform analysis on it.
+ */
+ORK_CLASS_AVAILABLE
+@interface ORKGoNoGoResult: ORKResult
+
+/**
+ The value of systemUptime (in NSProcessInfo) when the stimulus occurred.
+  */
+@property (nonatomic, assign) NSTimeInterval timestamp;
+
+/** 
+ Time from the stimulus occurred to the threshold being reached
+  */
+@property (nonatomic, assign) NSTimeInterval timeToThreshold;
+
+/**
+  YES if a go test and NO if a no go test
+  */
+@property (nonatomic, assign) BOOL go;
+
+/**
+  Set to YES if the incorrect response is given i.e shaken for no go test or not shaken for a go test
+  */
+@property (nonatomic, assign) BOOL incorrect;
+
+/**
+  An array of collected samples, in which each item is an `ORKGoNoGoSample`
+  object that represents an addition sample.
```

Comment:
consistency--either add '.' at the end of all the other property doc comments above, or remove this one.